ui-manchette-with-spacetimechart/src/helpers.ts

```diff
@@ -62,32 +56,28 @@ export const calcOperationalPointsToDisplay = (
   return result;
 };
 
-export const operationalPointsHeight = (
+export const calcOperationalPointsHeight = (
   operationalPoints: StyledOperationalPointType[],
-  height: number,
-  zoom: number,
-  isProportional: boolean
+  options: OperationalPointsOptions
```

Comment:
It's a suggestion to make the code more readable
```suggestion
  { height, isProportional, yZoom }: OperationalPointsOptions
```

Reply:
done